index.html

```diff
@@ -16,39 +16,161 @@ <h3><a href="#">Welcome</a></h3>
+          <button type="button" class="btn-bg-green">See Project</button>
+        </div>
+      </div>
+      <div class="project-list container flex">
+        <div class="project-about flex-column">
+          <h5 class="font-md">Multi-Post Stories <br>Gain+Glory</h5>
+          <ul class="flex">
+            <li class="li-padding bg-ash">Ruby on rails</li>
+            <li class="li-padding bg-ash">css</li>
+            <li class="li-padding bg-ash">JavScript</li>
+            <li class="li-padding bg-ash">html</li>
+          </ul>
+          <button type="button" class="btn-bg-green">See Project</button>
+        </div>
+      </div>
+      <div class="project-list container flex">
+        <div class="project-about flex-column">
+          <h5 class="font-md">Multi-Post Stories <br>Gain+Glory</h5>
+          <ul class="flex">
+            <li class="li-padding bg-ash">Ruby on rails</li>
+            <li class="li-padding bg-ash">css</li>
+            <li class="li-padding bg-ash">JavScript</li>
+            <li class="li-padding bg-ash">html</li>
+          </ul>
+          <button type="button" class="btn-bg-green">See Project</button>
+        </div>
+      </div>
+      <div class="project-list container flex">
+        <div class="project-about flex-column">
+          <h5 class="font-md">Multi-Post Stories <br>Gain+Glory</h5>
+          <ul class="flex">
+            <li class="li-padding bg-ash">Ruby on rails</li>
+            <li class="li-padding bg-ash">css</li>
+            <li class="li-padding bg-ash">JavScript</li>
+            <li class="li-padding bg-ash">html</li>
+          </ul>
+          <button type="button" class="btn-bg-green">See Project</button>
+        </div>
+      </div>
+      <div class="project-list container flex">
+        <div class="project-about flex-column">
+          <h5 class="font-md">Multi-Post Stories <br>Gain+Glory</h5>
+          <ul class="flex">
+            <li class="li-padding bg-ash">Ruby on rails</li>
+            <li class="li-padding bg-ash">css</li>
+            <li class="li-padding bg-ash">JavScript</li>
+            <li class="li-padding bg-ash">html</li>
+          </ul>
+          <button type="button" class="btn-bg-green">See Project</button>
+        </div>
+      </div>
+      <div class="project-list container flex">
+        <div class="project-about flex-column">
+          <h5 class="font-md">Multi-Post Stories <br>Gain+Glory</h5>
+          <ul class="flex">
+            <li class="li-padding bg-ash">Ruby on rails</li>
+            <li class="li-padding bg-ash">css</li>
+            <li class="li-padding bg-ash">JavScript</li>
+            <li class="li-padding bg-ash">html</li>
+          </ul>
+          <button type="button" class="btn-bg-green">See Project</button>
+        </div>
+      </div>
+    </section>
+    <section id="about-section" class="flex-column">
+      <div class="div">
+        <div class="abt-me container">
+          <h1>About me</h1>
```

Comment:
- [ Optional ] You should have only one `<h1>` in an HTML document for semantic reasons and you have already use one the headline section.

Reply:
Yeah, thanks